index.js

```diff
@@ -28,6 +28,12 @@ module.exports = function V(){
     };
   };
 
+
+  // Add custom validator
```

Comment:
this project is comment free so far, let's stick to the style and not introduce some